app/models/group_invite.rb

```diff
@@ -7,8 +7,16 @@ class GroupInvite < ApplicationRecord
 
   scope :unaccepted, -> { where(accepted: false) }
 
+  validate :user_not_already_in_group, on: :create
+
   private
 
+  def user_not_already_in_group
+    return unless group.users.where(email: email).exists?
+
+    errors.add(:group, "Can't send an invite to an existing member")
```

Comment:
Do we bubble this error message up to the user who is creating the invite?

Reply:
We don't, I'll add this. There are actually a few places in the application where we're not letting users know the success of failure of certain actions. I'll add an issue to cover that.